install.sh

```diff
@@ -320,6 +320,9 @@ __install() {
 	__install_mongodb & __load "[+] Installing MongoDB"
 
 	( # Build RITA
+		# Ensure go dep is installed
+		go get -u github.com/golang/dep/cmd/dep
```

Comment:
Dep's readme recommends installing from a versioned release for stability. We'd be versioning our version manager too which seems like a good idea. Thoughts on downloading it this way instead?

```
wget -nv -O $GOPATH/bin/dep https://github.com/golang/dep/releases/download/v0.3.2/dep-linux-amd64
chmod +x $GOPATH/bin/dep
```

Reply:
awesome idea

Reply:
Looks great to me too!